LICENSE

```diff
@@ -50,6 +50,11 @@ authors.
   - http://apache.mirrors.ovh.net/ftp.apache.org/dist/xerces/j/source/
   - [Apache Software License version 2.0][Apache-2.0]
 
+* juniversalchardet
+  - Java port of universalchardet from Mozilla
+  - https://code.google.com/archive/p/juniversalchardet/source/default/source
```

Comment:
It seems that the copyright holder is Netscape Communications Corporation and Contributors.